packages/commerce-sdk-react/package.json

```diff
@@ -59,14 +59,14 @@
     "@types/react-dom": "^18.2.1",
     "@types/react-helmet": "^6.1.6",
     "cross-env": "^5.2.1",
-    "internal-lib-build": "3.0.0",
+    "internal-lib-build": "3.0.1-preview.0",
     "jsonwebtoken": "^9.0.0",
     "nock": "^13.3.0",
     "nodemon": "^2.0.22",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-helmet": "^6.1.0",
-    "semver": "^7.3.8",
+    "semver": "^7.5.2",
```

Comment:
Although this `semver` upgrade is actually not necessary for this `commerce-sdk-react` package, we still need to do it to make sure all `semver` dependencies used by any package in the monorepo use the same version.